ModularTegustation/tegu_items/gadgets/unpowered.dm

```diff
@@ -143,21 +144,32 @@
 /obj/item/deepscanner/examine(mob/living/M)
 	. = ..()
 	if(deep_scan_log)
-		to_chat(M, "<span class='notice'>Previous Scan:[deep_scan_log]</span>")
+		to_chat(M, "<span class='notice'>Previous Scan:\n[deep_scan_log]</span>")
 
 /obj/item/deepscanner/attack(mob/living/M, mob/user)
-	user.visible_message("<span class='notice'>[user] takes a tool out of [src] and begins scanning [M].</span>", "<span class='notice'>You set down the deep scanner and begin scanning [M].</span>")
-	playsound(get_turf(M), 'sound/misc/box_deploy.ogg', 5, 0, 3)
-	if(!do_after(user, 2 SECONDS, target = user))
+	return
+
+/obj/item/deepscanner/afterattack(mob/living/target, mob/user, proximity_flag, click_parameters)
+	. = ..()
+	if(!istype(target))
+		return
+	Scan(target, user)
+
+/obj/item/deepscanner/proc/Scan(mob/living/target, mob/user)
+	if(!isanimal(target) && !ishuman(target))
 		return
-	if(ishuman(M))
-		var/mob/living/carbon/human/H = M
+	user.visible_message("<span class='notice'>[user] takes a tool out of [src] and begins scanning [target].</span>", "<span class='notice'>You set down the deep scanner and begin scanning [target].</span>")
```

Comment:
maybe remove the "You set down the deep scanner and begin scanning [src]" instead replacing it with just "you start scanning [src]"